hadoop-tools/hadoop-azure/src/test/java/org/apache/hadoop/fs/azurebfs/ITestAzureBlobFileSystemListStatus.java

```diff
@@ -55,7 +57,10 @@ public ITestAzureBlobFileSystemListStatus() throws Exception {
 
   @Test
   public void testListPath() throws Exception {
-    final AzureBlobFileSystem fs = getFileSystem();
+    Configuration config = new Configuration(this.getRawConfiguration());
+    config.set("fs.azure.list.max.results", "5000");
```

Comment:
Use the constant

Reply:
Is there tests with fs.azure.list.max.results with different values and asserting on the number of items returned.

Reply:
Replaced with const
Yes, [test](https://github.com/apache/hadoop/blob/c2cecfc9b951e21d38783227379bec63f34b7702/hadoop-tools/hadoop-azure/src/test/java/org/apache/hadoop/fs/azurebfs/ITestAbfsClient.java#L91) in ITestAbfsClient asserts return size for liststatus calls with different values of listMaxResults (set at AbfsConfig level)